js/scripts.js

```diff
@@ -0,0 +1,6 @@
+var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+var dinosaur = 'Triceratops';
+var UpperDinosaur = dinosaur.toUpperCase();
```

Comment:
first letter is reserved for constructor functions, variable names always start with small caps